samcli/lib/utils/subprocess_utils.py

```diff
@@ -92,10 +96,18 @@ def _print_loading_pattern():
                     # we read from subprocess stdout to avoid the deadlock process.wait function
                     # for more detail check this python bug https://bugs.python.org/issue1256
                     for line in process.stdout:
-                        decoded_line = _check_and_process_bytes(line)
+                        is_error = (
+                            IS_WINDOWS
+                            and len(line) >= len(TERRAFORM_ERROR_PREFIX)
+                            and line[0:4] == bytes(TERRAFORM_ERROR_PREFIX)
+                        )
```

Comment:
This utility module and function seems very generic, can we put terraform specific stuff behind a flag so that this will be only applied to the places where it is been called for terraform stuff?